<ui version="4.0" >
 <class>NetworkPage</class>
 <widget class="QWidget" name="NetworkPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>539</height>
   </rect>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::NoContextMenu</enum>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QCheckBox" name="chkUseProxy" >
     <property name="toolTip" >
      <string>Check to configure Tor to use a proxy to access the Internet</string>
     </property>
     <property name="text" >
      <string>I use a proxy to access the Internet</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpProxySettings" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>1</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>136</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>136</height>
      </size>
     </property>
     <property name="title" >
      <string>Proxy Settings</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="lblHttpProxyAddress" >
        <property name="text" >
         <string>HTTP Proxy:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lblHttpProxyUsername" >
        <property name="text" >
         <string>Username:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3" >
       <widget class="QCheckBox" name="chkProxyUseHttps" >
        <property name="text" >
         <string>Use this proxy for HTTPS also</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lineHttpProxyUsername" />
        </item>
        <item>
         <widget class="QLabel" name="lblHttpProxyPassword" >
          <property name="text" >
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineHttpProxyPassword" >
          <property name="echoMode" >
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1" colspan="4" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lineHttpProxyAddress" />
        </item>
        <item>
         <widget class="QLabel" name="lblHttpProxyPort" >
          <property name="text" >
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineHttpProxyPort" >
          <property name="maximumSize" >
           <size>
            <width>75</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maxLength" >
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="4" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkFascistFirewall" >
     <property name="contextMenuPolicy" >
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="toolTip" >
      <string>Check to configure Tor to only connect to relays using ports allowed by your firewall</string>
     </property>
     <property name="text" >
      <string>My firewall only lets me connect to certain ports</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpFirewallSettings" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>1</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>75</height>
      </size>
     </property>
     <property name="title" >
      <string>Firewall Settings</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="lblReachablePorts" >
        <property name="text" >
         <string>Allowed Ports:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineReachablePorts" >
        <property name="inputMask" >
         <string/>
        </property>
        <property name="text" >
         <string>80, 443</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkUseBridges" >
     <property name="toolTip" >
      <string>Check to configure Tor to use bridge relays to access the Tor network</string>
     </property>
     <property name="text" >
      <string>My ISP blocks connections to the Tor network</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpBridgeSettings" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="title" >
      <string>Tor Bridge Settings</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="lblNoBridgeSupport" >
        <property name="text" >
         <string>The Tor version you are currently running does not support bridges. &lt;br>Tor will still try to encrypt its directory connections.</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="lblAddBridge" >
          <property name="text" >
           <string>Add a Bridge:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lblHelpFindBridges" >
          <property name="cursor" >
           <cursor>13</cursor>
          </property>
          <property name="text" >
           <string>&lt;a href="bridges.finding">How do I find a bridge?&lt;/a></string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineBridge" >
            <property name="maxLength" >
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listBridges" >
            <property name="contextMenuPolicy" >
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="selectionMode" >
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QToolButton" name="btnAddBridge" >
            <property name="text" >
             <string/>
            </property>
            <property name="icon" >
             <iconset resource="../res/vidalia_common.qrc" >:/images/22x22/list-add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnRemoveBridge" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="toolTip" >
             <string>Remove the selected bridges from the list</string>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="icon" >
             <iconset resource="../res/vidalia_common.qrc" >:/images/22x22/list-remove.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnCopyBridge" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="toolTip" >
             <string>Copy the selected bridges to the clipboard</string>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="icon" >
             <iconset resource="../res/vidalia_common.qrc" >:/images/22x22/edit-copy.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkFascistFirewall</tabstop>
 </tabstops>
 <resources>
  <include location="../res/vidalia_common.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>chkFascistFirewall</sender>
   <signal>toggled(bool)</signal>
   <receiver>grpFirewallSettings</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>222</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel" >
     <x>222</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUseProxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>grpProxySettings</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>222</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel" >
     <x>222</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUseBridges</sender>
   <signal>toggled(bool)</signal>
   <receiver>grpBridgeSettings</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>283</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel" >
     <x>283</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
